inputs/select.vue

```diff
@@ -169,6 +169,7 @@
           }
 
           return {
+            ...option, // spread the whole object in case it has extra properties
             [this.valueKey]: this.equalLabelValueKeys ? option[label] : option[this.valueKey],
             [label]: option[label]
```

Comment:
Can we remove this if we're now unraveling option?

Reply:
oh ya good point - fixed now